src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/courtsentencing/CourtSentencingService.kt

```diff
@@ -860,6 +860,73 @@ class CourtSentencingService(
       telemetryClient.trackEvent("recall-inserted-ignored", telemetryMap, null)
     }
   }
+  suspend fun updateRecallSentences(recallUpdateEvent: RecallEvent) {
```

Comment:
this is nearly identical to the inserted above exact how it retrieves sentence Ids - might collapses these into a single function fi testing shows that can really do the same thing